diff --git a/Allwmake b/Allwmake
index d34b3665ec..1c2f9c6787 100755
--- a/Allwmake
+++ b/Allwmake
@@ -1,7 +1,7 @@
 #!/bin/sh
 # Run from OPENFOAM top-level directory only
 cd "${0%/*}" || exit
-wmake -check-dir "$WM_PROJECT_DIR" 2>/dev/null || {
+wmake -check-dir "$WM_PROJECT_DIR" || {
     echo "Error (${0##*/}) : not located in \$WM_PROJECT_DIR"
     echo "    Check your OpenFOAM environment and installation"
     exit 1
